src/main/java/duke/tasklist/task_types/Task.java

```diff
@@ -0,0 +1,47 @@
+package duke.tasklist.task_types;
+
+import java.io.Serializable;
+
+/**
+ * Represents a <code>task</code> object that contains a string holding the <code>name</code> of the
+ * task and the <code>status</code> of the task. Represents a <code>task</code> object that contains
+ * a string holding the <code>name</code> of the task and the <code>status</code> of the task.
+ * 
```

Comment:
Consider removing the blank line at line 9 to keep a consistent format for the header Javadoc of each class.